src/libs/actions/Report.ts

```diff
@@ -2530,10 +2530,11 @@ function navigateToMostRecentReport(currentReport: OnyxEntry<Report>) {
     const isChatThread = ReportUtils.isChatThread(currentReport);
     if (lastAccessedReportID) {
         const lastAccessedReportRoute = ROUTES.REPORT_WITH_ID.getRoute(lastAccessedReportID ?? '');
+        // We are using index 1 here because on index 0, we'll always have the bottomTabNavigator.
```

Comment:
> we'll always have the bottomTabNavigator

Why is this? Could you explain a little bit more. Thanks

Reply:
We use index  `=== 1` and not `=== 0` here because opening report from deep link will have two routes in navigation state `[home in LHN, report in center pane]` not only one route, so the first report route will be with index 1.

I think we can say it like this: 
```javascript
// The first report route should be with index 1 because report screen have two routes in navigation state `[home in LHN, report in center pane]`.
```

we may also need to rename it `isFirstRoute > isFirstReportRoute`

Reply:
Actualy this was a simpler reason than I initially thought 👍